src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("10.0.0.0")]
+[assembly: AssemblyVersion("10.1.0.0")]
 [assembly: AssemblyFileVersion("10.1.0.0")]
 
 [assembly: AssemblyConfiguration("")]
```

Comment:
please remove empty AssemblyConfiguration attributes. I know it has been there before, but if you are not going to use it, you can delete it.